tests/models/test_hooks.py

```diff
@@ -643,9 +643,6 @@ def test_trainer_model_hook_system_fit_no_val_and_resume(tmpdir):
         dict(name="train", args=(True,)),
         dict(name="on_train_dataloader"),
         dict(name="train_dataloader"),
-        # even though no validation runs, we initialize the val dataloader for properties like `num_val_batches`
```

Comment:
I remember noticing this and adding this comment because removing it wasn't straightforward. I'm glad we can do it now